Пагинация

alex912004 opened the pull request:
Добавлена пагинация

Commit:
jq grid pager

Deloff commented on src/PaginatorGridTrait.php:
```diff
@@ -0,0 +1,58 @@
+        if ($this->paginator === null) {
+            $adapter = $this->getAdapter();
+            if ($adapter instanceof \Zend\Paginator\Adapter\AdapterInterface) {
+                $this->paginator = new Paginator\Paginator($adapter);
```
Почему не через сервис локатор?

Commit:
jq grid pager

Commit:
Пагинация:
Cannot use Zend\Paginator\Adapter\AdapterInterface as AdapterInterface because the name is already in use

Deloff commented on src/Paginator/Paginator.php:
```diff
@@ -0,0 +1,78 @@
+     * @param array $possibleItemCountPerPage
+     * @return $this
+     */
+    public function setPossibleItemCountPerPage($possibleItemCountPerPage)
```
Проставить тип переменной

Commit:
Пагинация
